easybuild/easyconfigs/g/Gephi/Gephi-0.9.2.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'Gephi'
+version = '0.9.2'
+
+homepage = 'http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html'
+description = """ GTOOL is a program for transforming sets of genotype data for use with 
+the programs SNPTEST and IMPUTE.  """
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/gephi/gephi/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s-linux.tar.gz']
+
+checksums = ['e817c915e666634f924d9ab103da69e4e635bb8a47bcc5ad05c10b89d4cc23f3']
+
+dependencies = [('Java', '1.8.0_202')]
+sanity_check_paths = {
+    'files': ["bin/gephi", "bin/gephi64.exe", "etc/gephi.clusters"],
+    'dirs': ["bin", "etc", "gephi", "platform"]
+}
+
```

Comment:
Please add a sanity-check command, perhaps a simple `gephi -h`  if it exists

Reply:
Added in commit id 1464ae0